draft-ietf-quic-multipath.md

```diff
@@ -536,19 +529,18 @@ In non-final states, hosts have to track the following information.
 destination port) used by the endhost to send packets over the path.
 
 - Associated Destination Connection ID: The Connection ID used to send
-packets over the path.
+packets over the path. The endpoint relies on its sequence number to
+maintain a Destination Connection ID-specific packet number space for
+sending packets over this path. Packet number considerations as
+described in {{Section 12.3 of QUIC-TRANSPORT}} apply within the given
+path.
```

Comment:
I don't think we need this additional text.

You don't need the sequence number to identify the packet space; you can also just use the CID directly. You only need the sequence number really in control frames.

Also what consideration in RFC9000 do you want to point to?

Reply:
Indeed, let's remove that added text.